src/jit/lir.cpp

```diff
@@ -129,7 +129,14 @@ GenTree* LIR::Use::User() const
 //
 bool LIR::Use::IsValid() const
```

Comment:
The intent of the design is that the correctness checks run in the constructor and `IsValid` (which should really be called `IsInitialized`) ensures that the constructor has been run. By transitivity, the checks do not need to be re-run. This saves time in debug builds.